src/prefect/contrib/tasks/databricks/databricks_submitjob.py

```diff
@@ -360,3 +364,276 @@ def run(
         _handle_databricks_task_execution(self, hook, self.logger)
 
         return self.run_id
+
+
+class DatabricksRunNow(Task):
+    """
+    Runs an existing Spark job run to Databricks using the
+    `api/2.0/jobs/run-now
+    <https://docs.databricks.com/api/latest/jobs.html#run-now>`_
+    API endpoint.
+
+    There are two ways to instantiate this task.
+
+    In the first way, you can take the JSON payload that you typically use
+    to call the ``api/2.0/jobs/run-now`` endpoint and pass it directly
+    to our ``DatabricksRunNow`` task through the ``json`` parameter.
+    For example:
+
+    ```
+    json = {
+          "job_id": 42,
+          "notebook_params": {
+            "dry-run": "true",
+            "oldest-time-to-consider": "1457570074236"
+          }
+        }
+
+    conn = PrefectSecret('DATABRICKS_CONNECTION_STRING')
+    notebook_run = DatabricksRunNow(databricks_conn_string=conn, json=json)
+    ```
+
+    Another way to accomplish the same thing is to use the named parameters
+    of the ``DatabricksRunNow`` task directly. Note that there is exactly
+    one named parameter for each top level parameter in the ``run-now``
+    endpoint. In this method, your code would look like this:
+
+    ```
+    job_id=42
+
+    notebook_params = {
+        "dry-run": "true",
+        "oldest-time-to-consider": "1457570074236"
+    }
+
+    python_params = ["douglas adams", "42"]
+
+    spark_submit_params = ["--class", "org.apache.spark.examples.SparkPi"]
+
+    conn = PrefectSecret('DATABRICKS_CONNECTION_STRING')
+    notebook_run = DatabricksRunNow(
+        databricks_conn_string=conn,
+        notebook_params=notebook_params,
+        python_params=python_params,
+        spark_submit_params=spark_submit_params
+    )
+    ```
+
+    In the case where both the json parameter **AND** the named parameters
+    are provided, they will be merged together. If there are conflicts during the merge,
+    the named parameters will take precedence and override the top level `json` keys.
+
+    This task requires a Databricks connection to be specified as a Prefect secret and can
+    be passed to the task like so:
+
+    ```
+    from prefect.tasks.secrets import PrefectSecret
+    from prefect.contrib.tasks.databricks import DatabricksRunNow
+
+    with Flow('my flow') as flow:
+        conn = PrefectSecret('DATABRICKS_CONNECTION_STRING')
+        DatabricksRunNow(databricks_conn_string=conn, json=...)
+    ```
+
+    Currently the named parameters that ``DatabricksRunNow`` task supports are
+
+    - `job_id``
+    - `json``
+    - `notebook_params``
+    - `python_params``
+    - `spark_submit_params``
```

Comment:
```suggestion
    - `job_id`
    - `json`
    - `notebook_params`
    - `python_params`
    - `spark_submit_params`
```

Reply:
(the mix of single and double ticks was messing up formatting, having double elsewhere is fine)

Reply:
Sorry about that, will fix tomorrow and commit!